bin/buildkite

```diff
@@ -20,8 +20,12 @@ export COMMIT="${BUILDKITE_COMMIT}"
 export HEALTH_ENDPOINT='https://nps-collect.everydayhero.io/health'
 export JOB_ID="${BUILDKITE_JOB_ID}"
 export PUSH_TO_QUAY=false
-export JEFFREY_TOKEN_STAGING="${JEFFREY_TOKEN_STAGING}"
-export JEFFREY_TOKEN_PRODUCTION="${JEFFREY_TOKEN_PRODUCTION}"
+export JEFFREY_TOKEN_STAGING="${JEFFREY_TOKEN_STAGING:-}"
```

Comment:
Again, unsure - but i'm not used to seeing step related env variables within `buildkite` or `localkite` but rather within the related step (perhaps there is a reason here?)   I'm used to only seeing `plain-pipeline` related vars such as https://github.com/everydayhero/kanga/blob/master/bin/buildkite